ksql-engine/src/main/java/io/confluent/ksql/function/InternalFunctionRegistry.java

```diff
@@ -143,23 +142,11 @@ private void addStringFunctions() {
         "UCASE", UCaseKudf.class);
     addFunction(ucase);
 
-    KsqlFunction substring = new KsqlFunction(Schema.OPTIONAL_STRING_SCHEMA, Arrays.asList(
-        Schema.OPTIONAL_STRING_SCHEMA,
-        Schema.OPTIONAL_INT32_SCHEMA,
-        Schema.OPTIONAL_INT32_SCHEMA),
-        "SUBSTRING", SubstringKudf
-        .class);
-    addFunction(substring);
-    addFunction(new KsqlFunction(Schema.OPTIONAL_STRING_SCHEMA, Arrays.asList(
-        Schema.OPTIONAL_STRING_SCHEMA,
-        Schema.OPTIONAL_INT32_SCHEMA),
-        "SUBSTRING", SubstringKudf
-        .class));
 
-    KsqlFunction concat = new KsqlFunction(Schema.OPTIONAL_STRING_SCHEMA,
-        Arrays.asList(Schema.OPTIONAL_STRING_SCHEMA,
-            Schema.OPTIONAL_STRING_SCHEMA),
-        "CONCAT", ConcatKudf.class);
+    KsqlFunction concat = new KsqlFunction(Schema.STRING_SCHEMA, Arrays.asList(Schema.STRING_SCHEMA,
```

Comment:
Was it intentional to switch these back to non-optional? I ask as I know @hjafarpour has done work to switch things to optional.

Reply:
oops - thanks